### ISSUE TITLE
Timecodes for "ML Zoomcamp 8.8 - Adding More Layers"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#153]
Youtube video: https://www.youtube.com/watch?v=bSRRrorvAZs

**Comment by dimzachar** [comment]
0:00:00 - Lesson 8: Adding layers to neural network
0:01:48 - Adding inner layer for vector transformation.
0:03:46 - Activation and ReLU functions explained.
0:05:45 - Setting activation function and inner layer size.
0:07:35 - GPU utilization during model training displayed.
0:09:26 - Testing extra layer performance with dropout.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
